README.md

```diff
@@ -191,6 +191,9 @@ driver | 'compose' or 'espresso' | The name of the subdriver to use for elements
 
 [Jetpack Compose](https://developer.android.com/jetpack/compose) is Android’s modern toolkit for building native UI. Espresso driver supports basic interactions with Compose-based applications since version *1.46.0*.
 
+[Appium UiAutomator2 driver](https://github.com/appium/appium-uiautomator2-driver/) can interact with Jetpack Compose elements by setting [testTag](https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/package-summary#(androidx.compose.ui.Modifier).testTag(kotlin.String)) modifier attribute or the displayed text, but this Espresso driver allows you to access Jetpack Compose elements more directly.
```

Comment:
can interact with Jetpack Compose elements by setting -> allows to interact with Jetpack Compose elements via the accessibility layer by providing...